api/cmd/api/main.go

```diff
@@ -193,6 +197,22 @@ func main() {
 		fmt.Fprintf(os.Stderr, "invalid host argument: %q (valid hosts: localhost)\n", *hostF)
 	}
 
+	r := mux.NewRouter()
```

Comment:
without mux we can't do ?

Reply:
Nope we need mux as we have to create routes for auth